sensor/common/detector/enricher.go

```diff
@@ -397,3 +402,10 @@ func (e *enricher) outputChan() <-chan scanResult {
 func (e *enricher) stop() {
 	e.stopSig.Signal()
 }
+
+func compareImageName(x *storage.ImageName, y *storage.ImageName) bool {
```

Comment:
I think a small comment about why are we comparing it like this would be a good idea for the future us :)

Reply:
Good point. I can see the future us wondering why not using `proto.Equal` 😅